src/storage/database_sink.rs

```diff
@@ -613,6 +625,7 @@ impl<'s> DatabaseSink<'s> {
     fn insert_units(&mut self, abcd_data: &AbcdResult, id: u32) -> Result<(), Error> {
         let mut columns: Vec<String> = vec![self.database_settings.surrogate_key_column.clone()];
         columns.extend(self.unit_fields.iter().map(|field| field.hash.clone()));
+        columns.push("geom".to_owned());
```

Comment:
Again, you should just put it into the `vec![]` operation on line 626.

Reply:
no, geom has to be the last column because I first write all other columns and then the geom column using the lat/lon values gathered during the iteration over the columns